cfme/services/catalogs/ansible_catalog_item.py

```diff
@@ -16,6 +16,7 @@
 from utils.update import Updateable
 from utils.wait import wait_for
 from . import ServicesCatalogView
+from cfme.web_ui import mixins
```

Comment:
**REQURIED**
Old web ui elements shouldn't be used, please use `WidgetasticTaggable` mixin.